### ISSUE TITLE
ziprails dont work on 1.19.4

### ISSUE THREAD

**Issue by IbruuhI** [Gamemode4Dev/GM4_Datapacks#861]
**Describe the bug**
The Datapack wont work and the advancement has look like random numbers and letters

**To Reproduce**
1. Spawn a cat and '...'
2. Drop '....' onto the cat
3. Watch the cat eat '....'
4. cat.mcfunction stops working

**Expected behavior**
Describe what you expected to happen.

**Fix**
Outline how this bug could be fixed.

**Screenshots**
If applicable, add screenshots to help others understand your problem.

**Environment**
 - Minecraft version:1.19.4
 - Server version:1.19.4
 - Installed Datapacks:ziprails 1.19
 - Installed Mods:servux, headdrops, 1 player sleep

**Additional context**

**Comment by SpecialBuilder32** [comment]
Our datapacks currently do not support 1.19.4 as there are some significant technical changes we need to accommodate. An update is coming relatively soon however 

Your reproduction steps do confuse me though, as ziprails has nothing to do with cats?

**Comment by Bloo-dev** [comment]
I tested this in-game and **can not reproduce** the behaviour described by you.
I have:
1. Installed Ziprails on a fresh 1.19.4 world
2. Built a Zipline
3. Successfully used the Zipline

Here's a screenshot of my setup:
![image](https://user-images.githubusercontent.com/41843855/229368584-ae4298e2-849c-421c-9ec9-3dc3cca94a0f.png)

As for the broken text, that is another issue. I have created a bug report for it (#864), and we will hopefully get to fixing that soon.

I am closing this issue unless someone else can replicate ziprails as not working.
